### ISSUE TITLE
Can't load .json files because of deamdify bug

### ISSUE THREAD

**Issue by MrNice** [Famous/browserify-seed#7]
Hey @TheAlphaNerd,

When using famous *browserify-seed*, I couldn't require a json file into my project because famousify threw a parsing error. This turned out to be due to a bug in deamdify. There was a PR for fixing it, but deamdify appears to not be maintained.

Forked, merged PR, and used this one in my project:

https://github.com/MrNice/deamdify

which can be easily added to *browserify-seed* package.json:
```
"deamdify": "git://github.com/mrnice/deamdify.git#v0.1.2",
```

**Comment by TheAlphaNerd** [comment]
hmmmmm

So here is where I am at on this.  There is technically no reason that deamdify should be included in the repo anymore since the deamdification of famous is handled locally within the module.

I have just removed the dependency from the seed and pushed a new tag.

As an aside I think that the problem with deamdify not being properly maintained is a serious problem.  I noticed this thread https://github.com/jaredhanson/deamdify/issues/20

I bumped it and cc'd @jaredhanson about maintainability.  The biggest issue for the moment is maintaining the namespace on npm.  I chimed in and offered to take over maintainer-ship as I have experience with release management.  If that ends up working out and you are interested let me know and we can share the load

![share the load](http://www.quickmeme.com/img/c0/c00eb07c3b1d53d465a304dde5e7f43f4b9eeb74eb21bd709e16748897c564e7.jpg)

Thanks for pointing this out!
